app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -76,7 +76,6 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         webRenderComponents.notifyLaunchWithSafeIntent(safeIntent)
 
         lifecycle.addObserver(serviceLocator.engineViewCache)
-        lifecycle.addObserver(serviceLocator.pocketVideoFetchScheduler)
```

Comment:
It appears the onboarding screen still shows Pocket – can you file a follow-up to replace it, ping brampitoyo for the asset, and replace it once you get the asset?

Reply:
Filed https://github.com/mozilla-mobile/firefox-tv/issues/2999